### ISSUE TITLE
msmt17 trained with Resnet50

### ISSUE THREAD

**Issue by djidje** [KaiyangZhou/deep-person-reid#309]
Hi, in Model Zoo - Same Domain reID  https://kaiyangzhou.github.io/deep-person-reid/MODEL_ZOO.html
we should obtain as in first line Resnet50 is 63.2 R-1 (33.9 mAP) for msmt17.

When I download and resume training from the model I got 

```py 
Loading checkpoint from "/.../.../.../.../.../resnet50_msmt_xent.pth.tar"
Loaded model weights
Last epoch = 59
Last rank1 = 47.4%
````

Can you provide a model with the good Rank-1 accuracy please ?

The code to obtain this 

```py
import torchreid


batch_size = 32
lr = 0.0003

datamanager = torchreid.data.ImageDataManager(
        root='.../deep-person-reid/reid-data',
        sources='msmt17',
        targets='msmt17',
        width=128,
        batch_size_train=batch_size,
        batch_size_test=100,
        transforms=['random_flip', 'random_crop'],
        num_instances=4,
        train_sampler='RandomIdentitySampler',
        load_train_targets=False
)

model = torchreid.models.build_model(
        name='resnet50',
        num_classes=datamanager.num_train_pids,
        loss='triplet',
        pretrained=True
)

model = model.cuda()

optimizer = torchreid.optim.build_optimizer(
        model,
        optim='adam',
        lr=0.0003
)

scheduler = torchreid.optim.build_lr_scheduler(
        optimizer,
        lr_scheduler='single_step',
        stepsize=50
)

start_epoch = torchreid.utils.resume_from_checkpoint(
        '.../resnet50_msmt_xent.pth.tar',
        model,
        optimizer
)

model.cuda()

engine = torchreid.engine.ImageTripletEngine(
        datamanager,
        model,
        optimizer=optimizer,
        scheduler=scheduler,
        label_smooth=True,
        mmd_only=False
)

engine.run(
        test_only=True,
)

EDIT : after the testing results where those waiting for. Only an issue in the first print but the model is 63.2 Rank 1 thanks
